### ISSUE TITLE
No errors are shown when using undefined component

### ISSUE THREAD

**Issue by voronianski** [yannickcr/eslint-plugin-react#74]
No warning is shown when I'm using undeclared component:

```javascript
// ... is defined but never used
import {
    LoadingIndicator,
    // View
} from 'components';

...

render() {
    // no warnings..
    return (
        <View>
            <LoadingIndicator />
        </View>
    );
}
```

**Comment by yannickcr** [comment]
ESLint removed JSX support in `no-undef` since `v0.18.0`, you must now use the [`react/jsx-no-undef`](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-undef.md) rule.

**Comment by voronianski** [comment]
@yannickcr cool, that works!
